nautobot_circuit_maintenance/models.py

```diff
@@ -239,38 +235,30 @@ def to_csv(self):
 class NotificationSource(OrganizationalModel):
     """Model for Notification Source configuration."""
 
-    # Mark field as private so that it doesn't get included in ChangeLogging records!
-    _password = encrypt(models.CharField(max_length=100))
-    source_id = models.EmailField(max_length=100, unique=True, help_text="Email address used as identifier.")
-    url = models.CharField(max_length=200)
-    source_type = models.CharField(
-        default=NotificationSourceServerChoices.UNKNOWN,
-        max_length=50,
-        choices=NotificationSourceServerChoices,
-        null=True,
-        blank=True,
-        help_text="Type of source integration to retrieve the notifications.",
+    alias = models.CharField(
```

Comment:
I wonder if instead of `alias` we should use `name` (and/or `slug`?) to be more consistent with the rest of Nautobot? I don't feel strongly about this, just a thought.